server/graphql/v1/queries.js

```diff
@@ -936,6 +936,10 @@ const queries = {
         type: new GraphQLList(CountryISO),
         description: 'Limit the search to collectives belonging to these countries',
       },
+      tags: {
```

Comment:
According to https://github.com/opencollective/opencollective/issues/4378, shouldn't that be `tag`?

Reply:
Thanks for pointing it out. I've migrated this query to v2 so now this is no longer a problem. https://github.com/opencollective/opencollective-api/blob/3a48ed4a8f5df52d2358608faca041363ec5d1af/server/graphql/v2/query/collection/AccountsQuery.ts#L19